libs/crossplane/config.jsonnet

```diff
@@ -3,6 +3,12 @@ local config = import 'jsonnet/config.jsonnet';
 config.new(
   name='crossplane',
   specs=[
+    {
+      output: 'crossplane/1.10',
+      prefix: '^io\\.crossplane\\.(pkg|apiextensions|secrets|meta)\\..*',
+      crds: ['https://doc.crds.dev/raw/github.com/crossplane/crossplane@v1.10.1'],
+      localName: 'crossplane',
+    },
     {
       output: 'crossplane/1.3',
```

Comment:
Probably one for another PR, do we want to get rid of this `1.3` version too?

I imagine this was from a very old PoC